client/dashboard/plugins/manage/fields/update-available.ts

```diff
@@ -15,15 +15,17 @@ export const updateAvailableField: Field< PluginListRow > = {
 	enableHiding: false,
 	enableSorting: true,
 	elements: [
-		{ value: 2, label: __( 'Yes' ) },
-		{ value: 1, label: __( 'No' ) },
+		{ value: 2, label: __( 'Update available' ) },
```

Comment:
Suggestion: I think it'd be good to color code these states. cc: @lucasmendes-design